### ISSUE TITLE
Automatic cloud backup

### ISSUE THREAD

**Issue by meteficha** [openintents/safe#15]
It would be awesome if some form of automatic cloud backup was available. For example, saving the encrypted file on Google Drive. 

Is this feature something that would fit this app?

**Comment by friedger** [comment]
Yes, I would suggest to use the so-called document provider, that enables users to choose from any storage, whether cloud or not.

**Comment by friedger** [comment]
With the current release 1.4.3 the document provider is integrated and you can store your backup files in the cloud. It remains to do this automatically in a configurable way.
